sourcegit/sync.py

```diff
@@ -156,25 +176,14 @@ def sync(
                 )["id"]
                 logger.info(f"PR created: {dist_git_pr_id}")
 
-    @property
-    @lru_cache()
-    def pagure_read_token(self):
-        return os.environ["PAGURE_READ_TOKEN"]
-
-    @property
-    @lru_cache()
-    def pagure_edit_token(self):
-        """ this token is used to comment on pull requests """
-        # FIXME: make this more easier to be used -- no need for a dedicated token
-        return os.environ["PAGURE_EDIT_TOKEN"]
-
     @lru_cache()
     def get_repo(self, url, directory=None):
         if not directory:
             tempdir = tempfile.mkdtemp()
             self._tempdirs.append(tempdir)
             directory = tempdir
 
+        # TODO: optimize cloning: single branch and last n commits?
```

Comment:
I agree. That needs some optimisation. But we also need the origin branch/tag to compare with. Or not?

Reply:
since it's optimization, I wouldn't care about it too much now, this is just a note for us for future